deploy/olm-catalog/ibm-helm-repo-operator/3.6.0/operator.ibm.com_helmrepos_crd.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: helmrepos.operator.ibm.com
   labels:
     name: helmrepos.operator.ibm.com
-    app.kubernetes.io/instance: helm-repo
+    app.kubernetes.io/instance: helmrepos.operator.ibm.com
```

Comment:
did you mean to change this back to helmrepos.operator.ibm.com?

Reply:
darn..what's going on